articles/governance/blueprints/overview.md

```diff
@@ -80,14 +80,14 @@ as artifacts:
 
 ### Blueprint definition locations
 
-When creating a blueprint definition, you'll define where the blueprint is saved. Blueprints can be
+When creating a blueprint definition, you'll define where the blueprint is saved. Azure Blueprints can be
 saved to a [management group](../management-groups/overview.md) or subscription that you have
 **Contributor** access to. If the location is a management group, the blueprint is available to
 assign to any child subscription of that management group.
 
 ### Blueprint parameters
 
-Blueprints can pass parameters to either a policy/initiative or an Azure Resource Manager template.
+Azure Blueprints can pass parameters to either a policy/initiative or an Azure Resource Manager template.
```

Comment:
This entry is talking about the 'blueprints' object. It just happens to be at the start of the sentence, and thus capitalized. Please revert.